ignite/templates/module/create/files/base/x/{{moduleName}}/types/expected_keepers.go.plush

```diff
@@ -1,26 +1,88 @@
 package types
 
 import (
+	"cosmossdk.io/math"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/authz"
+	"github.com/cosmos/cosmos-sdk/x/feegrant"
+	"github.com/cosmos/cosmos-sdk/x/nft"
 )
 
 <%= for (dependency) in dependencies { %>
-<%= if (dependency.Name != "Bank" && dependency.Name != "Account") { %>
+<%= if (dependency.Name != "Account" && dependency.Name != "Bank" &&
+    dependency.Name != "Staking" && dependency.Name != "Slashing" &&
+    dependency.Name != "Distribution" && dependency.Name != "Mint" &&
+    dependency.Name != "Authz" && dependency.Name != "Feegrant" &&
+    dependency.Name != "Group" && dependency.Name != "Nft") { %>
 type <%= dependency.KeeperName() %> interface {
-	// Methods imported from <%= toLower(dependency.Name) %> should be defined here
+	// TODO Add methods imported from <%= toLower(dependency.Name) %> should be defined here
 }
 <% } %>
 <% } %>
 
-// AccountKeeper defines the expected account keeper used for simulations (noalias)
+// AccountKeeper defines the expected interface for the Account module.
 type AccountKeeper interface {
-	GetAccount(ctx sdk.Context, addr sdk.AccAddress) types.AccountI
+	GetAccount(sdk.Context, sdk.AccAddress) types.AccountI
 	// Methods imported from account should be defined here
 }
 
-// BankKeeper defines the expected interface needed to retrieve account balances.
+// BankKeeper defines the expected interface for the Bank module.
 type BankKeeper interface {
-	SpendableCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
+	SpendableCoins(sdk.Context, sdk.AccAddress) sdk.Coins
 	// Methods imported from bank should be defined here
 }
+
+// StakingKeeper defines the expected interface for the Staking module.
+type StakingKeeper interface {
+	TotalBondedTokens(sdk.Context) math.Int
+	// Methods imported from account should be defined here
+}
```

Comment:
Why are all the keepers being added instead of only the ones for the module dependencies?

Reply:
Will be addressed in #3625